book/00-prologue.html

```diff
@@ -182,67 +182,81 @@ <h2>A Brief History</h2>
-        expect from a general-purpose standard library. That's
-        because the standard library isn't a general-purpose tool;
-        it was developed for use in bootstrapping the compiler and
-        is purposefully kept small and simple.<idx>Core standard
-        library/development of</idx><idx>OCaml/Core standard
-        library</idx></p>
+        <h2>The <code>Base</code> Standard Library</h2>
+
+        <p>However good it is, a language on its own isn't enough. You
+          also need a set of libraries to build your applications
+          on. A common source of frustration for those learning OCaml
+          is that the standard library that ships with the compiler is
+          limited, covering only a subset of the functionality you
+          would expect from a general-purpose standard library. That's
+          because the standard library isn't really a general-purpose
+          tool; its fundamental role is in bootstrapping the compiler,
+          and has been purposefully kept small and simple.</p>
 
         <p>Happily, in the world of open source software, nothing
-        stops alternative libraries from being written to
-        supplement the compiler-supplied standard library, and this
-        is exactly what <code>Core</code> is.</p>
+          stops alternative libraries from being written to supplement
+          the compiler-supplied standard library.  <code>Base</code>
+          is an example of such a library, and it's the standard
+          library we'll use through most of this book.
+
+	  <idx>Base standard library</idx>
+	</p>
 
         <p>Jane Street, a company that has been using OCaml for
-        more than a decade, developed <code>Core</code> for its own
-        internal use, but designed it from the start with an eye
-        toward being a general-purpose standard library. Like the
-        OCaml language itself, <code>Core</code> is engineered with
-        correctness, reliability, and performance in mind.</p>
-
-        <p><code>Core</code> is distributed with syntax extensions
-        that provide useful new functionality to OCaml, and there
-        are additional libraries such as the <code>Async</code>
-        network communications library that extend the reach of
-        <code>Core</code> into building complex distributed
-        systems. All of these libraries are distributed under a
-        liberal Apache 2 license to permit free use in hobby,
-        academic, and commercial settings.</p>
+          more than 15 years, developed the code in <code>Base</code>
+          for its own internal use, but from the start designed it
+          with an eye toward being a general-purpose standard
+          library. Like the OCaml language itself, <code>Base</code>
+          is engineered with correctness, reliability, and performance
+          in mind.  It's also designed to be easy to install and
+          highly portable, available on every platform OCaml is,
```

Comment:
this `, available on every platform OCaml is,` reads oddly

Reply:
Good catch. Fixed.